### DISCUSSION TITLE
OpenCollective-China, Possible?

### DISCUSSION THREAD

**Discussion by Congyuwang** [opencollective/opencollective#6726] [General]
As far as I know, there is currently no platform in China similar to OpenCollective. The country has its own complexities regarding laws, regulations, and compliance. However, Chinese communities, including artists, activists, study groups, and community builders, are thriving in their own way but lack a management platform to offload their burdens and improve their funding sources. Many groups in China heavily rely on government funding and lack independence.

I wonder if anyone is interested in exploring and discussing the possibility of an 'OpenCollective-China' that can adapt to the particularities of Chinese law and the business environment, which can help the development of more independent Chinese communities. After all, the future is collective.

**Comment by Betree** [comment]
Related: https://github.com/opencollective/opencollective/issues/6584
